test/Microsoft.AspNet.Mvc.FunctionalTests/RequestServicesTest.cs

```diff
@@ -21,11 +21,10 @@ public RequestServicesTest(MvcTestFixture<RequestServicesWebSite.Startup> fixtur
         public HttpClient Client { get; }
 
         [Theory]
-        [InlineData("http://localhost/RequestScoped/FromController")]
-        [InlineData("http://localhost/Other/FromFilter")]
-        [InlineData("http://localhost/Other/FromView")]
-        [InlineData("http://localhost/Other/FromViewComponent")]
-        [InlineData("http://localhost/Other/FromActionArgument")]
+        [InlineData("http://localhost/RequestScopedService/FromFilter")]
+        [InlineData("http://localhost/RequestScopedService/FromView")]
+        [InlineData("http://localhost/RequestScopedService/FromViewComponent")]
+        [InlineData("http://localhost/RequestScopedService/FromActionArgument")]
```

Comment:
Good catch...and it was intentional as the scenario of `FromController` is not required as its covered by `FromActionArgument` now...we recently removed support for `FromServices` on a property which `FromController` was testing... https://github.com/aspnet/Mvc/pull/3671/files#diff-b2fc13d7d7ddd5d3ee5929cf0af23cc5L12